charts/kong/templates/_helpers.tpl

```diff
@@ -1142,21 +1142,6 @@ role sets used in the charts. Updating these requires separating out cluster
 resource roles into their separate templates.
 */}}
 {{- define "kong.kubernetesRBACRules" -}}
-- apiGroups:
-  - ""
-  resources:
-  - endpoints
-  verbs:
-  - list
-  - watch
-- apiGroups:
-  - ""
-  resources:
-  - endpoints/status
-  verbs:
-  - get
-  - patch
-  - update
```

Comment:
This will have to be made conditional on the KIC version that's being deployed similarly to https://github.com/Kong/charts/pull/796/files#diff-3e8a71242505abb72903a338eef1cdea2cd2c674f877c59378eff807951a1fa8R1491. Otherwise versions 2.9 and older will not get these permissions and this will effectively break KIC.